static/src/javascripts/projects/common/utils/picturefill.js

```diff
@@ -405,7 +405,7 @@ define([
                 candidates,
                 options = opt || {};
 
-            elements = options.elements || pf.getAllElements;
+            elements = options.elements || pf.getAllElements();
```

Comment:
[Updated this as per picturefill...](https://github.com/scottjehl/picturefill/blob/2.3/src/picturefill.js#L574)